src/lib/tools/workflow-steps.ts

```diff
@@ -77,6 +77,62 @@ const SHARED_PROMPTS = {
   `,
 };
 
+const contentTypes = z.enum([
+  "markdown",
+  "html",
+  "image",
+  "button_set",
+  "divider",
+  "partial",
+]);
+
+const contentBlockSchema = z.object({
+  type: contentTypes,
+});
```

Comment:
This is not a comprehensive schema for content blocks, but it demonstrates that the mcp server starts working once _some_ schema is provided.